### ISSUE TITLE
Hello, I used your deepDA code and then replaced the feature extraction network with shufflenetv2, but the accuracy rate was very low, only 0.4. But I use the shufflenetv2 network to classify the source domain dataset with very high accuracy, what is the problem

### ISSUE THREAD

**Issue by ww0929lib** [jindongwang/transferlearning#367]
**Before you open an issue**
If it's code running error, maybe you want to check the python or pytorch version before submitting an issue.

**Describe the bug**

**To Reproduce**
Where is this bug happen?

**Screenshots**

**Comment by jindongwang** [comment]
Our code is tuned using Resnet and Alexnet. You need to tune the hyperparameters using your own network.
